webui/react/src/components/kit/internal/Spinner/Spinner.tsx

```diff
@@ -37,7 +41,7 @@ const Spinner: React.FC<Props> = ({
             <Icon name="spinner" size={size} title="Spinner" />
           </div>
         }
-        spinning={!!spinning}
+        spinning={isSpinning}
```

Comment:
can we back out these changes? Adding another prop to control spinning here seems like a bad idea given that about half of the usages of spinner are currently broken. Instead, let's address that in a separate PR.